index.js

```diff
@@ -14,6 +14,7 @@ var characterMap = {
 	"Ầ": "A",
 	"Ằ": "A",
 	"Ȃ": "A",
+	"Á": "A",
```

Comment:
I just found that the capital `Á` actually is already supported - see L3. Just the capital though - the lowercase was not supported. Would you mind removing the capital one from the PR before we merge it?